src/strand.rs

```diff
@@ -53,10 +53,11 @@ impl Strand {
 impl PartialEq for Strand {
     /// Returns true if both are `Forward` or both are `Reverse`, otherwise returns false.
     fn eq(&self, other: &Strand) -> bool {
-        matches!(
-            (self, other),
-            (&Strand::Forward, &Strand::Forward) | (&Strand::Reverse, &Strand::Reverse)
-        )
+        match (self, other) {
+            (&Strand::Forward, &Strand::Forward) => true,
+            (&Strand::Reverse, &Strand::Reverse) => true,
+            _ => false,
+        }
```

Comment:
perhaps we should use
```rust
#[allow(clippy::match_like_matches_macro)]
```
here so it doesn't get "fixed" again in the future.

Reply:
Agreed- I've added it in the two offending locations.